BlobStorage/Advanced.cs

```diff
@@ -1277,150 +1054,100 @@ private static async Task ListBlobsHierarchicalListingAsync(CloudBlobContainer c
-                ICloudBlob blob = await container.GetBlobReferenceFromServerAsync(blobName);
-
-                // The previous call gets the blob's properties, so it's not necessary to call FetchAttributes
-                // to read a property.
-                Console.WriteLine("Blob {0} was last modified at {1} local time.", blobName,
-                    blob.Properties.LastModified.Value.LocalDateTime);
-            }
-            catch (StorageException e)
-            {
-                if (e.RequestInformation.HttpStatusCode == 404)
-                {
-                    Console.WriteLine("Blob {0} does not exist.", blobName);
-                    Console.WriteLine("Additional error information: " + e.Message);
-                }
-                else
-                {
-                    Console.WriteLine(e.Message);
-                    Console.ReadLine();
-                    throw;
-                }
+                Console.WriteLine(e.Message);
+                Console.ReadLine();
+                throw;
             }
-
             Console.WriteLine();
         }
 
         /// <summary>
         /// Creates the specified number of sequentially named block blobs, in a flat structure.
         /// </summary>
-        /// <param name="container">A CloudBlobContainer object.</param>
+        /// <param name="container">A BlobContainerClient object.</param>
         /// <param name="numberOfBlobs">The number of blobs to create.</param>
         /// <returns>A Task object.</returns>
-        private static async Task CreateSequentiallyNamedBlockBlobsAsync(CloudBlobContainer container, int numberOfBlobs)
+        private static async Task CreateSequentiallyNamedBlockBlobsAsync(BlobContainerClient container, int numberOfBlobs)
         {
             try
             {
-                CloudBlockBlob blob;
+                BlobClient blob;
                 string blobName = string.Empty;
-                MemoryStream msWrite;
 
                 for (int i = 1; i <= numberOfBlobs; i++)
                 {
                     // Format string for blob name.
                     blobName = i.ToString("00000") + ".txt";
 
                     // Get a reference to the blob.
-                    blob = container.GetBlockBlobReference(blobName);
+                    blob = container.GetBlobClient(blobName);
 
-                    // Set a property on the blob.
-                    blob.Properties.ContentType = "text/html";
+                    Dictionary<string, string> metadata = new Dictionary<string, string>();
 
                     // Set some metadata on the blob.
-                    blob.Metadata.Add("DateCreated", DateTime.UtcNow.ToLongDateString());
-                    blob.Metadata.Add("TimeCreated", DateTime.UtcNow.ToLongTimeString());
+                    metadata.Add("DateCreated", DateTime.UtcNow.ToLongDateString());
+                    metadata.Add("TimeCreated", DateTime.UtcNow.ToLongTimeString());
```

Comment:
Remove these 2 lines. Also, remove the "Set some metadata on the blob." comment above.
```suggestion
```